src/osx/Installer.Mac/uninstall.sh

```diff
@@ -3,6 +3,15 @@
 THISDIR="$( cd "$(dirname "$0")" ; pwd -P )"
 GCMBIN="$THISDIR/git-credential-manager-core"
 
+# Ensure we use the correct Homebrew prefix for
+# x64 and arm64 binaries.
+if [ ! -z "$HOMEBREW_PREFIX" ]
+then
+	INSTALLATIONPREFIX="$HOMEBREW_PREFIX"
+else
+	INSTALLATIONPREFIX=/usr/local
+fi
+
```

Comment:
This assumes that if we're being run outside of Homebrew (`$HOMEBREW_PREFIX` is unset) then we must be on an Intel Mac.

Could we not instead do a test for either:

1. The current CPU architecture, or
2. Where the uninstall.sh shell script is located? It should be co-located with the GCM exectuables.

Reply:
This information is also available via the `brew config` command. 

`brew config | grep HOMEBREW_PREFIX` will output `HOMEBREW_PREFIX: /opt/homebrew` on an M1 Mac

Reply:
My point was that this `uninstall.sh` script is also included and used for the normal `.pkg` installation route, without Homebrew at all.. so the script should also work for ARM64 Macs without Homebrew.

In this case, uninstall depends on where the `.pkg` picks to install when run on ARM64.

Reply:
Yep, I agree with this - I've got a fix in place and am planning to submit today.